vizro-ai/src/vizro_ai/dashboard/utils.py

```diff
@@ -61,3 +63,30 @@ def _register_data(all_df_metadata: AllDfMetadata) -> vm.Dashboard:
 
     for name, metadata in all_df_metadata.all_df_metadata.items():
         data_manager[name] = metadata.df
+
+
+def _dashboard_code(
+    dashboard: vm.Dashboard, extra_callable_defs: Optional[Set[str]] = None, extra_imports: Optional[Set[str]] = None
+) -> str:
+    """Get the code for the dashboard."""
+    return dashboard._to_python(extra_callable_defs=extra_callable_defs, extra_imports=extra_imports)
+
+
+def _process_to_set(list_of_list_of_dict):
```

Comment:
I would rename the argument so something more descriptive. Can we add typing?

This will not be needed anymore with the refactor, are we accounting for that?